priv/templates/channel/channel.ex

```diff
@@ -0,0 +1,18 @@
+defmodule <%= module %>Channel do
+  use <%= base %>.Web, :channel
+
+  def join("<%= plural %>:lobby", _auth_message, socket) do
```

Comment:
We should just have a single join with the _id of the module name, i.e.: `join("users:" <> user_id` ? Lobby is chat specific. Make sense?